src/main/java/com/devotedmc/ExilePearl/core/ExilePearlCore.java

```diff
@@ -461,9 +461,6 @@ public String getRealPlayerName(UUID uid) {
 	@SuppressWarnings("deprecation")
 	@Override
 	public UUID getUniqueId(String name) {
-		if (isNameLayerEnabled()) {
```

Comment:
I think there is or should be an equivalent "new NameAPI" api here, that would be preferred to use.

Reply:
Would below changes be adequate?

Reply:
Even though at the moment it resolves to that line of code, no, I'd recommend `NameAPI.getUUID(name)`

Reply:
(so basically just change to isArtemisEnabled() otherwise no change, haha, from what I see).